File: src/main/java/net/aspw/nightx/features/module/modules/movement/Bhop.java
package net.aspw.nightx.features.module.modules.movement;

import net.aspw.nightx.event.EventTarget;
import net.aspw.nightx.event.UpdateEvent;
import net.aspw.nightx.features.module.Module;
import net.aspw.nightx.features.module.ModuleCategory;
import net.aspw.nightx.features.module.ModuleInfo;

@ModuleInfo(name = "Bhop", category = ModuleCategory.MOVEMENT)
public class Bhop extends Module {
    @EventTarget
    public void onUpdate(final UpdateEvent event) {
        mc.thePlayer.jumpMovementFactor = 0.16f;
    }
}

File: src/main/java/net/aspw/nightx/features/module/modules/movement/Bhop.kt
package net.aspw.nightx.features.module.modules.movement

import net.aspw.nightx.event.EventTarget
import net.aspw.nightx.event.UpdateEvent
import net.aspw.nightx.features.module.Module
import net.aspw.nightx.features.module.ModuleCategory
import net.aspw.nightx.features.module.ModuleInfo

@ModuleInfo(name = "Bhop", category = ModuleCategory.MOVEMENT)
class Bhop : Module() {
    @EventTarget
    fun onUpdate(event: UpdateEvent?) {
        mc.thePlayer.jumpMovementFactor = 0.16f
    }
}